experimental/client-csharp/src/Wandb/Run.cs

```diff
@@ -1,5 +1,6 @@
 using WandbInternal;
 using Wandb.Internal;
+using Newtonsoft.Json;
```

Comment:
Opted into using this to ease downstream usage of the deserialized summary object, see Program.cs for an example.